### PR TITLE
TCK: Provide factory to obtain JsonValue from java.lang.Number #315

### PR DESCRIPTION
**Pull request by jbescos**
https://github.com/eclipse-ee4j/jsonp/issues/315

### CODE REVIEW

File: impl-tck/pom.xml [hunk 1]
```diff
@@ -33,8 +33,8 @@
     </modules>
 
     <properties>
-        <jsonp-api.version>2.0.2-SNAPSHOT</jsonp-api.version>
-        <jsonp-impl.version>2.0.2-SNAPSHOT</jsonp-impl.version>
+        <jsonp-api.version>2.0.1-SNAPSHOT</jsonp-api.version>
```
Comment (lukasj) [inline impl-tck/pom.xml:36]: Are these changes required/do they belong here?  The API should definitely be set to 2.1.0-SNAPSHOT if it is not already...

Comment (jbescos) [reply impl-tck/pom.xml:36]: The API version is still 2.0.1-SNAPSHOT. https://github.com/eclipse-ee4j/jsonp/blob/3c3b1b1d48f6bc1d3ccb5e2dda54c0b8939d3f81/api/pom.xml#L33

We can fix it later, when we have a 2.1 RC release. Right now we are in the middle of the changes, and it is good to be able to run the TCKs somehow locally.
